zerver/tests/test_management_commands.py

```diff
@@ -4,6 +4,9 @@
 from django.test import TestCase
 from django.conf import settings
 from django.core.management import call_command
+from StringIO import StringIO
```

Comment:
this import isn't python 3 compatible; you need to use `from six.moves import cStringIO as StringIO` (this is why the test failed)